client/packages/common/src/intl/locales/en/common.json

```diff
@@ -411,14 +414,14 @@
   "link.copy-to-clipboard": "Copy to Clipboard",
   "log.changed-from": "From",
   "log.changed-to": "to",
-  "log.invoice-created": "Shipment created",
-  "log.invoice-deleted": "Shipment deleted",
-  "log.invoice-number-allocated": "Shipment number allocated",
-  "log.invoice-status-allocated": "Shipment allocated",
-  "log.invoice-status-delivered": "Shipment delivered",
-  "log.invoice-status-picked": "Shipment picked",
-  "log.invoice-status-shipped": "Shipment shipped",
-  "log.invoice-status-verified": "Shipment verified",
+  "log.invoice-created": "Created",
+  "log.invoice-deleted": "Deleted",
+  "log.invoice-number-allocated": "Invoice number allocated",
```

Comment:
I'm concerned that the term `Invoice` could be confusing. I would keep as `Shipment` or to be consistent with the minimalist approach, it would be `Number allocated`

Reply:
`Shipment number allocated` would be inaccurate when used in returns - can go with `Number allocated`, that just seems super vague to me? But do you think that would make more sense to users than `invoice number`?